AspNetCoreApplication/Repositories/BookCategoryRepository.cs

```diff
@@ -48,5 +55,17 @@ public async Task Delete(int bookId, int categoryId)
             await dataContext.SaveChangesAsync();
         }
 
+        public async Task CheckRole(int id)
```

Comment:
Ở đây là class BookCategory nên những tên gọi id này phải rõ ràng để tránh confuse

```suggestion
        public async Task CheckRole(int bookId)
```

Reply:
done